ui/package.json

```diff
@@ -17,7 +17,7 @@
     "lint": "yarn eslint src --ext .ts,.tsx",
     "codegen": "yarn run codegen-clean && yarn run codegen-swagger && yarn run notebooks-codegen-swagger && yarn run tanagra-codegen-openapi",
     "codegen-clean": "rm -rf src/generated src/notebooks-generated src/tanagra-generated",
-    "codegen-swagger": "java -jar ../aou-utils/swagger-codegen-cli.jar generate --lang typescript-fetch --input-spec ../api/src/main/resources/workbench-api.yaml --output src/generated/fetch",
```

Comment:
Configuration information can be found here:
https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/typescript-fetch.md